components/components/blocks/branding/logo.svg

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="205px" height="61px" viewBox="0 0 205 61" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
-    <title>Logo</title>
-    <desc>Logo for Umami theme.</desc>
+<svg viewBox="0 0 205 61" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
As the search icon, I would pass the new logo icon through a compressor like https://jakearchibald.github.io/svgomg/. Anyway, as I said in Slack I'd try to add this logo as a real vector instead of having the <image> tag inside. But I'm fine if we come back later when @kjay provides us with the new one.

Reply:
@ckrina as we sad in Slack ththis will be a followup issue: https://github.com/lauriii/umami/issues/75